e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeLiveIngesterE2eTest.java

```diff
@@ -0,0 +1,174 @@
+
+import org.alfresco.hxi_connector.common.test.docker.repository.AlfrescoRepositoryContainer;
+import org.alfresco.hxi_connector.common.test.docker.util.DockerContainers;
+import org.alfresco.hxi_connector.common.test.util.RetryUtils;
+import org.alfresco.hxi_connector.e2e_test.util.client.RepositoryClient;
+import org.alfresco.hxi_connector.e2e_test.util.client.model.Node;
+
+@Slf4j
+@Testcontainers
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SuppressWarnings("PMD.FieldNamingConventions")
+public class UpdateNodeLiveIngesterE2eTest
+{
+    private static final Network network = Network.newNetwork();
+    @Container
+    private static final PostgreSQLContainer<?> postgres = DockerContainers.createPostgresContainerWithin(network);
+    @Container
+    private static final GenericContainer<?> activemq = DockerContainers.createActiveMqContainerWithin(network);
+    @Container
+    private static final GenericContainer<?> sfs = DockerContainers.createSfsContainerWithin(network);
+    @Container
+    private static final GenericContainer<?> transformCore = DockerContainers.createTransformCoreAioContainerWithin(network)
+            .dependsOn(activemq);
+    @Container
+    private static final GenericContainer<?> transformRouter = DockerContainers.createTransformRouterContainerWithin(network)
+            .dependsOn(activemq, transformCore);
+    @Container
+    private static final WireMockContainer hxInsightMock = DockerContainers.createWireMockContainerWithin(network)
+            .withFileSystemBind("src/test/resources/wiremock/hxinsight", "/home/wiremock", BindMode.READ_ONLY);
+    @Container
+    private static final LocalStackContainer awsMock = DockerContainers.createLocalStackContainerWithin(network);
+    @Container
+    private static final AlfrescoRepositoryContainer repository = createRepositoryContainer()
+            .dependsOn(postgres, activemq, transformCore, transformRouter, sfs);
+    @Container
+    private static final GenericContainer<?> liveIngester = createLiveIngesterContainer().dependsOn(activemq, hxInsightMock, repository);
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+    private static final String ALLOW_ACCESS_PROPERTY = "ALLOW_ACCESS";
+    private static final String DENY_ACCESS_PROPERTY = "DENY_ACCESS";
+    public static final String GROUP_EVERYONE = "GROUP_EVERYONE";
+    public static final String ALICE = "abeecher";
+    public static final String MIKE = "mjackson";
+    protected RepositoryClient repositoryClient;
+
+    @BeforeAll
+    @SneakyThrows
+    public void beforeAll()
+    {
+        repositoryClient = new RepositoryClient(repository.getBaseUrl(), ADMIN_USER);
+        WireMock.configureFor(hxInsightMock.getHost(), hxInsightMock.getPort());
+    }
+
+    @AfterEach
+    void tearDown()
+    {
+        WireMock.reset();
+    }
+
+    @Test
+    @SuppressWarnings({"PMD.JUnitTestsShouldIncludeAssert"})
+    final void testCreateNodeWithDefaultPermissions()
```

Comment:
I'm not getting why for this test was created separate class - it couldn't be placed in [UpdateNodeE2eTest](https://github.com/Alfresco/hxinsight-connector/blob/4a19a9bbec1743f475d9e7016c4f2a043bd39c3b/e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java)?

Reply:
I would say that we should split e2e tests because in fact we have 3 separate flows:

bulk ingester -> live ingester -> hxi

repo -> live ingester -> hxi

hxi -> prediction applier -> repo

and imo mixing them is not a good approach (eventually we could mix first two but definitely not 3-rd one)

Reply:
For component "endpoint2enpoint" testing we have those: https://github.com/Alfresco/hxinsight-connector/tree/ff1d1f9ba5391db64ae854153643d726a24bdaf3/live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository

Reply:
moved!